src/schema/server_settings.py

```diff
@@ -173,280 +221,182 @@ def setting_schema(cls, name: str) -> Dict[str, Any]:
     )
 
-    image: Optional[str] = Field(
-        default=None,
-        meta_setting={
-            "name": "image",
-            "label": "Image",
-            "description": "Select an image to represent this adventure.",
-            "type": "image",
-            "default": "",
-            "required": True,
-            "suggestOutputType": "image",
-        },
+    image: Optional[str] = SettingField(
+        default="",
+        label="Image",
+        description="Select an image to represent this adventure.",
+        type="image",
+        required=True,
     )
     """For use on the profile marketing page and also during 'Magic Create' mode in the editor."""
 
-    short_description: Optional[str] = Field(
-        default="An amazing story of exploration.",
-        meta_setting={
-            "name": "adventure_short_description",
-            "label": "Short Description",
-            "description": "A catchy one-liner to help your adventure stand out in the discover page",
-            "type": "text",
-            "default": "",
-            "required": True,
-            "suggestOutputType": "short_description",
-        },
+    short_description: Optional[str] = SettingField(
+        default="",
+        label="Short Description",
+        description="A catchy one-liner to help your adventure stand out in the discover page",
+        type="text",
+        required=True,
     )
 
-    description: Optional[str] = Field(
-        default="An amazing story of exploration.",
-        meta_setting={
-            "name": "description",
-            "label": "Description",
-            "description": "A longer description of this adventure. Go into detail!",
-            "type": "textarea",
-            "default": "",
-            "required": True,
-            "suggestOutputType": "description",
-        },
+    description: Optional[str] = SettingField(
+        default="",
+        label="Description",
+        description="A longer description of this adventure. Go into detail!",
+        type="textarea",
+        required=True,
     )
     """For use on the profile marketing page and also during 'Magic Create' mode in the editor."""
 
-    tags: Optional[List[Optional[str]]] = Field(
+    tags: Optional[List[Optional[str]]] = SettingField(
         default=None,
-        meta_setting={
-            "name": "tags",
-            "label": "Tags",
-            "description": "A list of short string tags.",
-            "type": "tag-list",
-            "listof": "text",
-        },
+        label="Tags",
+        description="A list of short string tags.",
+        type="tag-list",
+        listof="text"
     )
     """For use on the profile marketing page and also during 'Magic Create' mode in the editor."""
 
+    # TODO what to do about this one?  It's not currently exported.
```

Comment:
We could def export it here -- IIRC it's in the schema file but probably just wasn't yet ported to the new Field style of describing it

Reply:
hm, I'd have expected it to show up in the endpoint result diff I've been doing at some point if it had dropped out but I don't remember seeing that